packages/web/src/components/Button/_Button.scss

```diff
@@ -11,6 +11,7 @@
     align-items: center;
     padding: theme.$padding-y theme.$padding-x;
     text-align: center;
+    text-decoration: none;
```

Comment:
Should this be here? Maybe having a `.Button_link` class will be better. What do you think @crishpeen?

Reply:
although it should rather be a modifier `Button--link`, but in this case it seemed useless to me. It is only always set that the text in the button will not be underlined, which will always be the case.

Reply:
I would keep it here as suggested in your PR. It's easier to use and we do not differ if you use `<button>` or `<a>` in button implementation in the web package.